MatrixSDK/Threads/MXThread.swift

```diff
@@ -29,7 +29,14 @@ public class MXThread: NSObject {
     /// Identifier of the room that the thread is in.
     public let roomId: String
     
+    private var _liveTimeline: MXEventTimeline?
```

Comment:
Why using `_` prefix? seems that `liveTimeline` can be used here.

Reply:
You right, redacted.